compiler/rustc_mir/src/transform/check_consts/ops.rs

```diff
@@ -33,7 +33,12 @@ pub fn non_const<O: NonConstOp>(ccx: &ConstCx<'_, '_>, op: O, span: Span) -> boo
                         concat!(r#"#[rustc_const_unstable(feature = "...", issue = "...")]"#, '\n').to_owned(),
                         Applicability::HasPlaceholders,
                     )
-                    .note("otherwise `#[allow_internal_unstable]` can be used to bypass stability checks")
+                    .span_suggestion(
+                        ccx.body.span,
```

Comment:
this should only be `ccx.body.span.lo()` I think? Or we may even the `ccx.tcx.def_span(ccx.def_id.to_def_id()) so it's inserted before the item and not as part of the body.

Reply:
Yeah, the `def_span` one seems most correct.

Reply:
`def_span` seems to point before the block as well. I ended up using `fn_sig().span.shrink_to_lo()`, which should do the right thing.